### ISSUE TITLE
Some reminder about the Crello dataset

### ISSUE THREAD

**Issue by skyerhxx** [CyberAgentAILab/canvas-vae#3]
Firstly, thanks for the authors' good dataset. 

Just a reminder, in both v1 and v2, there are some rendered elements parsed from the .tfrecord files whose height or width are not greater than 0.
In my parsing process using the python code provided by the author, I met the error like this
![image](https://user-images.githubusercontent.com/31229906/171017851-7d4eed48-0642-432f-8a11-09f7f82273dd.png)
After removing some error cases, the number of remaining imgs is
![image](https://user-images.githubusercontent.com/31229906/171018274-f02028d1-6a29-4260-9753-97664308415e.png)

Besides, there are some elements whose top or left coordinate is smaller than 0. I think it is because the rendered element is larger than its space in the image like Fig.2. My approach is to set left/top coordinate = 0 if they are < 0 like Fig.3 (the resulted appearance may be a little bit different from the original img, but it does not influence the classification of the element)
![image](https://user-images.githubusercontent.com/31229906/171019399-d9cd453a-8bf3-45bb-8f53-2f8742c85818.png)

**Comment by sukritiverma1996** [comment]
I also encountered width, height > 1 in some samples
So I'm capping that at 1.0

**Comment by skyerhxx** [comment]
There are also some incomplete cases in textElement, like
![image](https://user-images.githubusercontent.com/31229906/173759430-1672ac87-f31b-447c-be11-54356bd8f920.png)

**Comment by sukritiverma1996** [comment]
Thanks for that
Do you happen to know what "maskElement" means?
Some elements are tagged as "maskElement"

**Comment by skyerhxx** [comment]
no idea, very confused.........

**Comment by kyamagu** [comment]
Thanks for reporting this. Here are some clarifications.

We have tried our best to reverse-engineer undocumented data structure from the source website, but there are still erroneous elements and fields that we did not handle. For example, the width or height might be zero. We leave them as is.
`maskElement` seems to be equivalent to a cropped image element.

Any element can have an out-of-canvas position as an element can be bigger than the canvas (then cropped). There is no preprocessing yet to crop such elements.

As documented in the README, text rendering is far from perfect. For example, a newline character is not supported.
